index.html

```diff
@@ -1,1046 +1,1080 @@
+      expressing Verifiable Credentials and Verifiable Presentations
+      as described in the [[VC-DATA-MODEL]], using approaches
+      described by the COSE and JOSE working groups at IETF. This
+      includes JWTs [[RFC7519]], JWS [[RFC7515]], and COSE [[RFC8152]]
+      and provides an approach using well defined content types
+      [[RFC6838]] to identify that the type of data being secured
+      conforms with the [[VC-DATA-MODEL]].
+    </p>
+    <p>
+      JSON Web Signatures (JWS) [[RFC7515]] provide a standardized
+      mechanism for digitally signing JSON data. It provides a means
+      to ensure the integrity, authenticity, and non-repudiation of
+      the information contained in a JSON object. JWS enables secure
+      communication and trust between parties by adding a
+      cryptographic signature to a payload. This signature is
+      generated using a private key and can be verified using the
+      corresponding public key. JWS enables content integrity, authentication and non repudiation, 
+      protecting data from unauthorized modifications. The
+      versatility and wide adoption of JWS make it an essential
+      component for verifying the authenticity and integrity of
+      data in various applications, including
+      authentication, authorization, and secure data exchange. These
+      properties make JWS especially well suited to securing the
+      JSON-LD [[VC-DATA-MODEL]] and through this to provide a means of
+      supporting the three party model (issuer, verifier, holder) for
+      exchange of credentials enabled in verifiable credenials.
+    </p>
+    <p>
+      JSON Web Token (JWT) [[RFC7519]] is a widely-used means of
+      expressing
+      <a data-cite="VC-DATA-MODEL#claims">claims</a>, represented in JSON, to be transferred
+      between two parties. Providing a representation of the
+      Verifiable Credentials Data Model for JWT allows existing
+      systems and libraries to participate in the ecosystem described
+      in Section <a
+      data-cite="VC-DATA-MODEL#ecosystem-overview">ecosystem
+      overview</a>. A JWT encodes a set of
+      <a data-cite="VC-DATA-MODEL#claims">claims</a> as a JSON object
+      that is contained in a JSON Web Signature (JWS) [[RFC7515]]
+      and/or JSON Web Encryption (JWE) [[?RFC7516]]. For this
+      specification, the use of JWE is out of scope.
+      <p class="issue">
+      The working group is discussing how we might comment on recent work, that does make
+      use of encryption in <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-3.2">OpenID for Verifiable Presentations</a>.
+      </p> 
+    </p>
+    <p>
+      CBOR Object Signing and Encryption (COSE) [[RFC8152]] is a
+      specification that defines a framework for signing and
+      encrypting CBOR (Concise Binary Object Representation) [RFC8949]
+      data structures. COSE provides a standardized way to secure the
```

Comment:
CBOR Object Signing and Encryption (COSE) [[RFC8152]] is a
      specification that defines a framework for signing and
      encrypting CBOR (Concise Binary Object Representation) [RFC8949]
      data structures.
->
CBOR Object Signing and Encryption (COSE) [[RFC8152]] is a
      specification that defines a framework for representing signed and
      encrypted data using (Concise Binary Object Representation) [RFC8949]
      data structures.

Reply:
It's CWT that defines using COSE to sign CBOR claims - not COSE.  If we want to talk about using COSE to secure CBOR data structures then reference CWT [RFC 8392].